src/org/traccar/helper/Checksum.java

```diff
@@ -243,4 +243,12 @@ public static long luhn(long imei) {
         return (10 - (checksum % 10)) % 10;
     }
 
+    public static int modulo256(byte[] bytes) {
```

Comment:
I though about the name and I think it makes sense to rename it to `modulo8`. Other functions here are named with number of bits.